integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkEnvIT.java

```diff
@@ -19,16 +19,16 @@
 import org.apache.spark.sql.SparkSession;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.BeforeAll;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-/** Setup Hive, Gravitino, Spark, Metalake environment to execute SparkSQL. */
```

Comment:
`Hive` seems ok for hive related enviroment setup by hive container.

Reply:
fixed.